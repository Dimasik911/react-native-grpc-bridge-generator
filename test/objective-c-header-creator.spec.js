/* eslint-env jest */

import iosHeader from '../src/objective-c-header-creator'

it('creates header output', () => {
  const rsp = iosHeader('FunService')

  expect(rsp).toMatchSnapshot()
})
