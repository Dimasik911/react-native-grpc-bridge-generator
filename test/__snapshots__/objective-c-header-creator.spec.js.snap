// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates header output 1`] = `
"@import Foundation;
#import <React/RCTBridgeModule.h>

@interface FunService: NSObject <RCTBridgeModule>

@end"
`;
